django/contrib/admin/static/admin/css/forms.css

```
@@ -76,6 +76,24 @@ form ul.inline li {
     padding-right: 7px;
 }
 
+/* FIELDSETS */
+
+fieldset legend {
+    inline-size: 100%;
+    padding: 0;
+}
+
+fieldset h3 {
+    /* The border exists for line continuity + forced-colors mode */
```

I think we really want to refer to "high-constrast"? IIUC "forced-colors" is just the mechanism for forcing a particular color mode in a browser? See the description [here](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors).

```suggestion
    /* The border exists for line continuity and high-contrast mode. */
```

---

I removed the comment as I think we don't need it but I understand your point 👍